backend/chainlit/server.py

```diff
@@ -255,61 +258,22 @@ def replace_between_tags(text: str, start_tag: str, end_tag: str, replacement: s
     return re.sub(pattern, start_tag + replacement + end_tag, text, flags=re.DOTALL)
 
 
-def get_html_template():
-    PLACEHOLDER = "<!-- TAG INJECTION PLACEHOLDER -->"
-    JS_PLACEHOLDER = "<!-- JS INJECTION PLACEHOLDER -->"
-    CSS_PLACEHOLDER = "<!-- CSS INJECTION PLACEHOLDER -->"
-
-    default_url = "https://github.com/Chainlit/chainlit"
-    default_meta_image_url = (
-        "https://chainlit-cloud.s3.eu-west-3.amazonaws.com/logo/chainlit_banner.png"
-    )
-    url = config.ui.github or default_url
+def get_html_template(request: Request):
+    default_url = "https://app.purposepilot.co/"
+    default_meta_image_url = "https://app.purposepilot.co/logo?theme=light"
```

Comment:
We want our code to be as close to chainlit's main fork: this way, we can easily update our fork without merge conflicts. 

There is no functional benefit whatsoever in setting custom defaults, especially considering that we'll likely do more than just Purpose Pilot in the future!

Rather, we should just add a single parameter allowing arbitrary header overrides _in a second pull request_.